core/capabilities/ccip/delegate.go

```diff
@@ -39,7 +40,6 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/keys/p2pkey"
 	"github.com/smartcontractkit/chainlink/v2/core/services/ocr2"
 	"github.com/smartcontractkit/chainlink/v2/core/services/ocrcommon"
-	p2ptypes "github.com/smartcontractkit/chainlink/v2/core/services/p2p/types"
```

Comment:
We often import `github.com/smartcontractkit/chainlink/v2/core/services/p2p/types` just for aliased things in `ragep2p/types`.